packages/Webkul/Product/src/Database/Migrations/2018_12_06_185202_create_product_flat_table.php

```diff
@@ -26,7 +26,7 @@ public function up()
 
             $table->decimal('price', 12, 4)->nullable();
             $table->decimal('cost', 12, 4)->nullable();
-            $table->boolean('special_price')->nullable();
+            $table->decimal('special_price', 12, 4)->nullable();
```

Comment:
First this runs,
then later https://github.com/bagisto/bagisto/pull/4467/files#diff-a223ebdcbd152cb1b7e0f62a142e6b1ae7a30e0fb50368839494af1b409f6c5bL19 runs.
Does this make sense to you? 
It's redundant and can cause problems.

Reply:
You have done changes in the existing migration please create the new one.